test/ruby-test-mode-test.el

```diff
@@ -30,11 +27,12 @@
   )
 
 (ert-deftest ruby-test-specification-filename ()
-  (should (equal "spec/models/file_spec.rb" (ruby-test-specification-filename "app/models/file.rb")))
-  (should (equal "spec/controllers/file_spec.rb" (ruby-test-specification-filename "app/controllers/file.rb")))
-  (should (equal "spec/helpers/file_spec.rb" (ruby-test-specification-filename "app/helpers/file.rb")))
-  (should (equal "spec/views/posts/new.html.erb_spec.rb" (ruby-test-specification-filename "app/views/posts/new.html.erb")))
-  (should (equal "spec/services/some_service/file_spec.rb" (ruby-test-specification-filename "app/services/some_service/file.rb")))
-  (should (equal "spec/lib/something/file_spec.rb" (ruby-test-specification-filename "lib/something/file.rb")))
-  (should (equal "spec/lib/some_lib/file_spec.rb" (ruby-test-specification-filename "lib/some_lib/file.rb")))
-  (should (equal "something/file_spec.rb" (ruby-test-specification-filename "something/file.rb"))))
+  (should (equal "project/spec/models/file_spec.rb" (ruby-test-specification-filename "project/app/models/file.rb")))
+  (should (equal "project/spec/controllers/file_spec.rb" (ruby-test-specification-filename "project/app/controllers/file.rb")))
+  (should (equal "project/spec/helpers/file_spec.rb" (ruby-test-specification-filename "project/app/helpers/file.rb")))
+  (should (equal "project/spec/views/posts/new.html.erb_spec.rb" (ruby-test-specification-filename "project/app/views/posts/new.html.erb")))
+  (should (equal "project/spec/services/some_service/file_spec.rb" (ruby-test-specification-filename "project/app/services/some_service/file.rb")))
+  (should (equal "project/spec/something/file_spec.rb" (ruby-test-specification-filename "project/lib/something/file.rb")))
+  (should (equal "project/spec/some_lib/file_spec.rb" (ruby-test-specification-filename "project/lib/some_lib/file.rb")))
+  (should (equal "project/something/file_spec.rb" (ruby-test-specification-filename "project/something/file.rb")))
+  )
```

Comment:
Lines too long as well.

Also maybe switch the argument order of those equal calls? I think I would prefer this:
``` elisp
(should (equal (ruby-test-specification-filename "project/app/models/file.rb")
               "project/spec/models/file_spec.rb"))
(should (equal (ruby-test-specification-filename "project/app/controllers/file.rb")
               "project/spec/controllers/file_spec.rb"))
(should (equal (ruby-test-specification-filename "project/app/helpers/file.rb")
               "project/spec/helpers/file_spec.rb"))
(should (equal (ruby-test-specification-filename "project/app/views/posts/new.html.erb")
               "project/spec/views/posts/new.html.erb_spec.rb"))
(should (equal (ruby-test-specification-filename "project/app/services/some_service/file.rb")
               "project/spec/services/some_service/file_spec.rb"))
(should (equal (ruby-test-specification-filename "project/lib/something/file.rb")
               "project/spec/something/file_spec.rb"))
(should (equal (ruby-test-specification-filename "project/lib/some_lib/file.rb")
               "project/spec/some_lib/file_spec.rb"))
(should (equal (ruby-test-specification-filename "project/something/file.rb")
               "project/something/file_spec.rb"))
```